[2025-09-09T14:52:58.074+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db manual__2025-09-05T00:11:34.425860+00:00 [queued]>
[2025-09-09T14:52:58.290+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db manual__2025-09-05T00:11:34.425860+00:00 [queued]>
[2025-09-09T14:52:58.316+0000] {taskinstance.py:2170} INFO - Starting attempt 4 of 5
[2025-09-09T14:52:58.555+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_sql_db> on 2025-09-05 00:11:34.425860+00:00
[2025-09-09T14:52:58.626+0000] {standard_task_runner.py:60} INFO - Started process 363 to run task
[2025-09-09T14:52:58.701+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'banvic_etl', 'extract_sql_db', 'manual__2025-09-05T00:11:34.425860+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/bavic_dag.py', '--cfg-path', '/tmp/tmpljql1lbs']
[2025-09-09T14:52:58.745+0000] {standard_task_runner.py:88} INFO - Job 99: Subtask extract_sql_db
[2025-09-09T14:53:00.657+0000] {task_command.py:423} INFO - Running <TaskInstance: banvic_etl.extract_sql_db manual__2025-09-05T00:11:34.425860+00:00 [running]> on host 9948058d4968
[2025-09-09T14:53:02.241+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Paulo' AIRFLOW_CTX_DAG_ID='banvic_etl' AIRFLOW_CTX_TASK_ID='extract_sql_db' AIRFLOW_CTX_EXECUTION_DATE='2025-09-05T00:11:34.425860+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-05T00:11:34.425860+00:00'
[2025-09-09T14:53:03.063+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-09T14:53:03.795+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-09T14:53:04.053+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-09T14:53:04.453+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-09T14:53:04.745+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=banvic_etl, task_id=extract_sql_db, execution_date=20250905T001134, start_date=20250909T145258, end_date=20250909T145304
[2025-09-09T14:53:04.937+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-09T14:53:05.137+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
